icon/iiss/icstate/networkvalue.go

```diff
@@ -378,7 +378,7 @@ func (s *State) GetNetworkInfoInJSON() (map[string]interface{}, error) {
 	jso["consistentValidationPenaltySlashRatio"] = s.GetConsistentValidationPenaltySlashRatio()
 	jso["unstakeSlotMax"] = s.GetUnstakeSlotMax()
 	jso["delegationSlotMax"] = s.GetDelegationSlotMax()
-	jso["proposalNonVotePenaltySlashRatio"] = s.GetNonvotedPenaltySlashRatio()
+	jso["proposalNonVotePenaltySlashRatio"] = s.GetNonVotedPenaltySlashRatio()
```

Comment:
Which one is correct?
proposalNonVotePenaltySlashRatio vs proposalNonVotedPenaltySlashRatio